jsonschema/src/resolver.rs

```diff
@@ -55,25 +144,19 @@ impl Resolver {
                 }
                 match self.schemas.get(url_str) {
                     Some(value) => Ok(value.clone()),
-                    None => match url.scheme() {
-                        "http" | "https" => {
-                            #[cfg(any(feature = "reqwest", test))]
-                            {
-                                let response = reqwest::blocking::get(url.as_str())?;
-                                let document: Value = response.json()?;
-                                let arcdoc = Arc::new(document);
-                                self.store
-                                    .write()
-                                    .insert(url_str.to_string(), arcdoc.clone());
-                                Ok(arcdoc)
-                            }
-                            #[cfg(not(any(feature = "reqwest", test)))]
-                            panic!("trying to resolve an http(s), but reqwest support has not been included");
+                    None => {
+                        if let Some(s) = self.store.read().get(url_str) {
```

Comment:
Hm, it should not be necessary, as `store` is checked a few lines above

Reply:
Indeed, I removed the earlier check instead as it requires a lock, not that it would matter too much.